modules/dqt/static/schema.yml

```diff
@@ -0,0 +1,346 @@
+              properties:
+                destination:
+                 type: string
+      responses:
+        '204':
+          description: File moved successfully
+  /dqt/search:
+    post:
+      description: Execute the query
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Query'
+      responses:
+        '201':
+          description: Results created
+          headers:
+            Location:
+              schema:
+                type: string
+              description: Created results url
+                 
+components:
+  schemas:
+    Queries:
+      type: object
+      properties:
+        history:
+          type: array
+          items:
+            $ref: '#/components/schemas/Query'
+        saved:
+          type: array
+          items:
+            $ref: '#/components/schemas/SavedQuery'
+        shared:
+          type: array
+          items:
+            $ref: '#/components/schemas/SavedQuery'
+    
+    SavedQuery:
+      type: object
+      required:
+        - name
+        - shared
+      properties:
+        id:
+          type: integer
+        name:
+          type: string
+        shared:
+          type: boolean
+        creatorUsername:
+          type: string
+          description: This value will be ignored in POST and PATCH requests.
+        creationTimestamp: 
+          type: string
+          description: This value will be ignored in POST and PATCH requests.
+        query:
+          $ref: '#/components/schemas/Query'
+    
+    Query:
+      type: object
+      required:
+        - fields
+        - filters
+      properties:
+        fields:
+          $ref: '#/components/schemas/Fields'
+        filters:
+          $ref: '#/components/schemas/Filters'
+          
+    Fields:
+      anyOf:
+        - $ref: '#/components/schemas/Field'
+        
+    Field:
+      type: object
+      required:
+        - category
+        - field
+      properties:
+        category:
+          type: string
+        field:
+          type: string
+        visits:
+          type: array
+          items:
+            type: string
+      
+    Filters:
+      oneOf:
+        - $ref: '#/components/schemas/Group'
+        - $ref: '#/components/schemas/Filter'
+      discriminator:
+        propertyName: type
+  
+    Group:
+      type: object
+      required:
+        - type
+        - operator
+        - items
+      properties:
+        type:
+          type: string
+          enum:
+            - 'group'
+        operator:
+          type: string
+          enum:
+            - 'AND'
+            - 'OR'
+        items:
+          type: array
+          items:
+            $ref: '#/components/schemas/Filters'
+
+      example:
+        ex1: 
+          type: 'group'
+          operator: 'AND'
+          items:
+            - type: object
+              operator: 'equals'
+              category: 'demographics'
```

Comment:
I think once #6936 goes in this (and other places referring to "category"/"field name") are also going to have to know what module the data comes from in some way.